rbd/namespace_nautilus_test.go

```diff
@@ -42,6 +42,10 @@ func TestNamespace(t *testing.T) {
 		assert.Error(t, err)
 		_, err = NamespaceExists(ioctx, "")
 		assert.Error(t, err)
+
+		// NamespaceList.
```

Comment:
This is a redundant comment.

Reply:
I have added function name as a comment for each function block in this test block, same has been done for NamsespaceList(). So its not as such redundant but in the same fashion as the code above this.

Reply:
OK, I'll live with it for now. :-)